src/styles/globals.css

```
@@ -127,3 +127,56 @@
+thead {
+  /* Using a color similar to code block backgrounds */
+  @apply bg-neutral-800; /* Darker background for header */
+}
+
+th {
+  @apply border border-neutral-600 px-4 py-2 text-left font-medium text-neutral-200; /* Lighter text for header */
+}
+
+tbody td { /* Increase specificity */
+  @apply border border-neutral-700 px-4 py-2;
+  color: theme('colors.neutral.100') !important; /* Force apply theme color */
```

Biome (1.9.4)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)